index.html

```diff
@@ -1382,6 +1388,13 @@ <h3>Related Media Object (TTML)</h3>
           provides the media timeline and is the source of the <a>main programme audio</a>,
           and any visual timing references needed when adjusting timings relevant to the video image,
           such as for lip synchronization.</p>
+          <div class="example note">
+          <p>A document instance can indicate the location of the programme acting
```

Comment:
```suggestion
          <p>A document instance can identify the programme acting
```

It might just hold a workflow-specific identifier rather than a globally defined resource locator.